sci-wcalc/parallel_rc_s2p.man

.\" Copyright (c), 2005 Dan McMahill
.\" Do not edit this directly.  Edit the XML source file instead
.\"

.TH parallel_rc_s2p "" "" "Wcalc" "Wcalc Commands"
.B parallel_rc_s2p
- Convert Series RC Network into Equivalent Parallel RC Network

.SH SYNOPSIS

[] = 
parallel_rc_s2p(Cs, Rs, Qs, freq, flag, Cs, Rs, Qs, Cp, Rp, Qp)


.SH INPUT PARAMETERS

.TP 10
Cs
: Series capacitance (Farads)
.TP 10
Rs
: Series resistance (Ohms)
.TP 10
Qs
: Series quality factor
.TP 10
freq
: Analysis frequency (Hertz)
.TP 10
flag
: indicates if Qs or Rs should be used for the loss input
    
.RS
.TP 10
flag=0
: Qs will be calculated from Rs.
.TP 10
flag=1
: Rs will be calculated from Qs.
.RE

.TP 10
Cs
: Series capacitance (Farads)
.TP 10
Rs
: Series resistance (Ohms)
.TP 10
Qs
: Series quality factor
.TP 10
Cp
: Parallel capacitance (Farads)
.TP 10
Rp
: Parallel resistance (Ohms)
.TP 10
Qp
: Parallel quality factor

.SH OUTPUT VALUES

.SH DESCRIPTION

Function to calculate an equivalent parallel RC network from an input
series RC network.

.SH EXAMPLE
.nf

.TP
 // 10 pF
Cs=10e-12;
.TP
 // 0.5 Ohms
Rs=0.5;
.TP
 // This value will be replaced
Qs=100;
.TP
 // 100 MHz operation
f=100e6;
.TP
 // Use Rs (and calculate Qs)
flag=0;
[Cs, Rs, Qs, Cp, Rp, Qp] = parallel_rc_s2p(Cs, Rs,
  Qs, f, flag);
disp(sprintf('Equivalent parallel C = %g pF', Cp/1e-12));
disp(sprintf('Equivalent parallel R = %g kOhm', Rp/1e3));
disp(sprintf('Quality factor         =%g',Qp));
.fi
.SH SEE ALSO
stdio-wcalc(1),
air_coil_calc(), air_coil_syn(), coax_calc(), bars_calc(), coax_syn(), coplanar_calc(), coplanar_syn(), coupled_microstrip_calc(), coupled_microstrip_syn(), coupled_stripline_calc(), coupled_stripline_syn(), ic_microstrip_calc(), ic_microstrip_syn(), microstrip_calc(), microstrip_syn(), parallel_rc_p2s(), parallel_rc_s2p(), stripline_calc(), stripline_syn()
.SH AUTHOR

Dan McMahill

.SH BUGS

None known


NEWS
$Id: NEWS,v 1.10 2009/02/28 03:26:18 dan Exp $

Wcalc -- history of user-visible changes. 

Copyright (c) 1998, 1999, 2000, 2001, 2002, 2003, 2004, 2005
2006, 2007, 2008, 2009 Dan McMahill

Please report bugs on the sourceforge project page.

===============
Version 1.1
===============

Added series/parallel RC equivalent circuit calculator.

Added series/parallel RL equivalent circuit calculator.

Added self and mutual inductance of two rectangular bars.

Added coplanar waveguide model.

Added coupled stripline model.

Corrected the Q calculations for low frequency (not skindepth) region
in air core inductors.  Also corrected Q calculation for conductors
other than copper.

Corrected the calculation of incremental conductance for the
microstrip model.

Corrected a bug in the calculation of conductor losses in the
microstrip model.

Added/enabled code for calculating conductor losses in the coupled
microstrip model.

When compiling with gtk-2.10 or newer, use gtkPrint for printing.
This gives a more professional/standard print dialog and also enables
printing under windows.

Added a preliminary Dutch translation.

Converted the figures to all use solid grayscale fills instead of
pattern fills as the former prints better.

Several updates to the build system to moderize it a bit.  Uses
AM_CONDITIONALS more appropriately, uses AC_PROG_LIBTOOL instead of
AM_PROG_LIBTOOL, etc.

Update the win32 build script to generate an installer that works
correctly under vista.

Add .wc file associations under vista.

Update the octave build infrastructure to work with octave-3.0 and
newer where mex support is built into mkoctfile instead of a
standalone mex shell script.

Switch to GPL licensing.

Create several utility functions to make it faster to build new gtk
forms.  This is only visible to developers.

Improved autogen.sh to check versions of several of the auto* tools to
help make it easier to diagnose any issues which may come up.

Repair compilation with the SunPRO compilers.


===============
Version 1.0
===============
Replaced the alert dialog box in the gtk version with a log
window.  This prevents a problem when many alerts are generated
and the user is presented with many many alert windows.

Added win32 support along with a build script for a windows installer.

Fixed a bug which gave lots of error/warning messages in the cgi
frontend for coupled microstrip.

===============
Version 0.9
===============
First public release.

===============
Version 0.1 
===============
Initial tagged version.  This does good analysis and synthesis
of microstrip.  Considered stable for this purpose.  However,
open/save/print and units other than mils are not supported yet.



octave-wcalc/parallel_rc_s2p.m
%PARALLEL_RC_S2P  - Convert Series RC Network into Equivalent Parallel RC Network 
%  
%  SYNOPSIS
%  
%  [] = parallel_rc_s2p(Cs, Rs, Qs, freq, flag, Cs, Rs, Qs, Cp, Rp, Qp)
%  
%  PARAMETERS
%  
%  INPUT PARAMETERS
%  
%    * Cs : Series capacitance (Farads)
%    * Rs : Series resistance (Ohms)
%    * Qs : Series quality factor
%    * freq : Analysis frequency (Hertz)
%    * flag : indicates if Qs or Rs should be used for the loss input
%        + flag=0 : Qs will be calculated from Rs.
%        + flag=1 : Rs will be calculated from Qs.
%    * Cs : Series capacitance (Farads)
%    * Rs : Series resistance (Ohms)
%    * Qs : Series quality factor
%    * Cp : Parallel capacitance (Farads)
%    * Rp : Parallel resistance (Ohms)
%    * Qp : Parallel quality factor
%  
%  OUTPUT VALUES
%  
%  DESCRIPTION
%  
%  Function to calculate an equivalent parallel RC network from an input series RC
%  network.
%  
%  EXAMPLE
%  
%   % 10 pF
%  Cs=10e-12;
%   % 0.5 Ohms
%  Rs=0.5;
%   % This value will be replaced
%  Qs=100;
%   % 100 MHz operation
%  f=100e6;
%   % Use Rs (and calculate Qs)
%  flag=0;
%  [Cs, Rs, Qs, Cp, Rp, Qp] = parallel_rc_s2p(Cs, Rs,
%    Qs, f, flag);
%  disp(sprintf('Equivalent parallel C = %g pF', Cp/1e-12));
%  disp(sprintf('Equivalent parallel R = %g kOhm', Rp/1e3));
%  disp(sprintf('Quality factor         =%g',Qp));
%  
%  SEE ALSO
%  
%  air_coil_calc
%  air_coil_syn
%  coax_calc
%  bars_calc
%  coax_syn
%  coplanar_calc
%  coplanar_syn
%  coupled_microstrip_calc
%  coupled_microstrip_syn
%  coupled_stripline_calc
%  coupled_stripline_syn
%  ic_microstrip_calc
%  ic_microstrip_syn
%  microstrip_calc
%  microstrip_syn
%  parallel_rc_p2s
%  parallel_rc_s2p
%  parallel_rl_p2s
%  parallel_rl_s2p
%  stripline_calc
%  stripline_syn
%  
%  AUTHOR
%  
%  Dan McMahill
%  
%  BUGS
%  
%  None known
%  
%  Please report any bugs on the Wcalc Sourceforge Project Page
%  


% Copyright (c) 2001-2009 Dan McMahill
% All rights reserved.
%
% This program is free software; you can redistribute it and/or modify
% it under the terms of the GNU General Public License as published by
% the Free Software Foundation; version 2 of the License.
% 
% This program is distributed in the hope that it will be useful,
% but WITHOUT ANY WARRANTY; without even the implied warranty of
% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
% GNU General Public License for more details.
% 
% You should have received a copy of the GNU General Public License
% along with this program; if not, write to the Free Software
% Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
%
